src/pages/encrypt/index.js

```diff
@@ -4,50 +4,42 @@ import OptimizelySubdomain from '../../components/optimizelysubdomain.js';
-          <meta property="og:url" content="https://advocacy.mozilla.org/encrypt/"/>
-          <meta property="og:description" content="Encryption needs you! Sign Mozilla's pledge to stand up for encryption and a more secure Web at advocacy.mozilla.org/encrypt"/>
-          <meta property="og:image" content="https://advocacy.mozilla.org/assets/img/share/Encryption-facebook-b5-06.png" />
+          <meta property="og:title" content={this.props.metaTitle}/>
+          <meta property="og:site_name" content={this.props.metaSiteName}/>
+          <meta property="og:url" content={this.props.metaUrl}/>
+          <meta property="og:description" content={this.props.metaDesc}/>
+          <meta property="description" content={this.props.metaDesc}/>
+          <meta property="og:image" content={this.props.metaImage} />
 
           <meta name="twitter:card" content="summary_large_image" />
           <meta name="twitter:site" content="@mozilla" />
-          <meta name="twitter:title" content="Encryption Matters" />
-          <meta name="twitter:description" content="Encryption needs you! Sign Mozilla's pledge to stand up for encryption & a more secure Web at mzl.la/encrypt" />
-          <meta name="twitter:image" content="https://advocacy.mozilla.org/assets/img/share/Encryption-twitter-b5-06.png" />
-          <meta description="Your online privacy depends on encryption. Learn more about how encryption works, why it's essential to a strong Web, and why it's worth protecting." />
+          <meta name="twitter:title" content={this.props.metaTitle} />
+          <meta name="twitter:description" content={this.props.metaDesc} />
+          <meta name="twitter:image" content={this.props.twitterImage} />
+          <meta description={this.props.desc} />
 
           <link rel="preconnect" href="https://www.google-analytics.com" />
           <link rel="preconnect" href="https://fonts.gstatic.com" />
-          <link rel="stylesheet" type="text/css" href="/build/encrypt.css"/>
-          <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Fira+Sans:300i,300,400,600"/>
+          <link rel="stylesheet" type="text/css" href="/build/main.css"/>
+          <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Fira+Sans:300,300i,400i,400,600"/>
           <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
-          <title>Learn about encryption - Mozilla Advocacy</title>
+          <title>{this.props.title}</title>
           <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="/assets/favicon/apple-touch-icon-180x180.png"/>
           <link rel="icon" type="image/png" sizes="196x196" href="/assets/favicon/favicon-196x196.png"/>
           <link rel="shortcut icon" href="/assets/favicon.ico"/>
           <OptimizelySubdomain/>
           <Optimizely/>
-          <script dangerouslySetInnerHTML={{__html: ga}}></script>
```

Comment:
ga is handled in client.js because that way react router page views get tracked.